spock-specs/src/test/groovy/org/spockframework/smoke/extension/StepwiseExtension.groovy

```diff
@@ -92,7 +91,117 @@ class Foo extends Specification {
     then:
     result.testsSucceededCount == 1
     result.testsFailedCount == 0
+    result.testsSkippedCount == 0
+  }
+
+  def "[feature] basic usage"() {
+    runner.throwFailure = false
+
+    when:
+    def result = runner.runWithImports("""
+class Foo extends Specification {
+  @Stepwise
+  def step1() {
+    expect: count != 3
+    where: count << (1..5)
+  }
+}
+    """)
+
+    then:
+    result.testsStartedCount == 3 + 1
```

Comment:
You should assert the skip message at least once.

Reply:
Why? The message can change. I am asserting on the number of skipped iterations already. For the spec-level annotation, there is no assertion on the skip message either. I find it interesting that not for the first time, you are requiring something from me which the original code does not have. But anyway, I can comply with that, if you are asking in the spirit of continuous improvement, wishing me to apply the boyscout rule "leave the campground cleaner than you found it". In that case, I will also add a corresponding check in for the spec-based annotation.

Reply:
Done in 9ef415c.